permissions/plugin-git-bisect.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "git-bisect"
+paths:
+- "org/jenkins-ci/plugins/git-bisect"
```

Comment:
Does not match pom.xml. Please read this repository's readme and adapt either this file, or the pom.xml.

Reply:
Which field does not match? I'm probably looking at the wrong field..

Reply:
You've specified a `groupId` of `git`, which overrides the parent `groupId` of `org.jenkins-ci.plugins`.

The simplest fix would be to delete the `<groupId>git</groupId>` line in your POM.

Reply:
Thanks. I had a feeling that it shouldn't be there, had no idea what it does.

Iv'e removed it.